src/hmr/hmr-helper.js

```diff
@@ -28,6 +28,15 @@ function removeCurrentFiles(fileList) {
     }
   });
 }
+function clearIntervals(){
+      // Get a reference to the last interval + 1
+      const interval_id = window.setInterval(function(){}, Number.MAX_SAFE_INTEGER);
+
+      // Clear any timeout/interval up to that id
+      for (let i = 1; i < interval_id; i++) {
```

Comment:
It looks very brute. I wonder what the side effects could be here. I think we should also clear the interval that we created here.
```suggestion
      for (let i = 1; i <= interval_id; i++) {
```